UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -161,7 +161,7 @@ public Acceleration(double meterspersecondsquared)
         /// <summary>
         ///     The <see cref="BaseDimensions" /> of this quantity.
         /// </summary>
-        public static BaseDimensions BaseDimensions => new BaseDimensions(1, 0, -2, 0, 0, 0, 0);
+        public static BaseDimensions BaseDimensions = new BaseDimensions(1, 0, -2, 0, 0, 0, 0);
```

Comment:
You need to add either `readonly` to make that a field, or `{ get; } = ` to make it a statically initialized getter property. I prefer the latter, mostly because it can be included in an interface or base type later with less changes.